ee/tabby-ui/app/auth/signup/components/admin-register.tsx

```diff
@@ -0,0 +1,84 @@
+'use client'
+
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+
+import { cn } from '@/lib/utils'
+import { Button } from '@/components/ui/button'
+
+import { UserAuthForm } from './user-register-form'
+
+import './admin-register.css'
+
+export default function AdminRegister() {
+  const router = useRouter()
+  const [step, setStep] = useState(1)
+
+  return (
+    <div
+      className={cn('admin-register-wrap w-[550px]', {
+        'translate-y-1/3': step === 1,
+        '-translate-y-10': step === 2,
+        '-translate-y-20': step === 3
+      })}
+    >
+      <div
+        className={cn('border-l border-foreground py-8 pl-12', {
+          'step-mask': step !== 1,
+          remote: step > 2
+        })}
+      >
```

Comment:
Please extract a helper function (e.g `<RegisterStep>`), to make it easier to add / modify steps